src/app/hm-admin/services/models/PagedCollection.ts

```diff
@@ -0,0 +1,22 @@
+export class PagedCollection {
+
+  public context;
+  public id;
+  public type;
+  public firstPage;
+  public itemsPerPage;
+  public lastPage;
+  public members;
+  public totalItems;
+
+  constructor(properties: any = {}, definition: any = {}) {
```

Comment:
Yes. Btw, this PagedCollection should be in the List PR, not here, but we will complete it later